templates/admin/http_stubs/includes/fieldset_http_stub.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'admin/http_stubs/includes/fieldset.html' %}
+{% load stub_tags %}
+{% block writable_field %}
+    {{ field.field }}
+    {# added request url for path field(issue #35) #}
+    {% if field.field.name == 'path' %}<div class="readonly"><div id="full-path">{% absolute field.field.value %}</div><div id="path-copied-alert">Скопировано!</div></div>{% endif %}
```

Comment:
Cyrillic characters still remain. Also, HTML doesn't impose the requirement to have everything in one extremely long line.
HTML can be broken at almost any boundary, this can greatly enhance readability of this line.

Reply:
Changed. Thanks!